src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -158,6 +158,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
+        $choiceTranslationDomain = $options['choice_translation_domain'];
+        $translationDomain = $options['translation_domain'];
+
+        if ($view->parent) {
+            if (null === $choiceTranslationDomain) {
+                $choiceTranslationDomain = $view->parent->vars['translation_domain'];
```

Comment:
that is already done in the lazy option. but if  both translation_domain and choice_translation_domain is null, it must read it from the parent. otherwise it would be a bc break.